tests/scripts/all.sh

```diff
@@ -914,6 +924,20 @@ component_build_no_sockets () {
     make CC=gcc CFLAGS='-Werror -Wall -Wextra -O0 -std=c99 -pedantic' lib
 }
 
+component_test_memory_buffer_allocator () {
+    msg "build: default config with memory buffer allocator enabled"
+    scripts/config.pl set MBEDTLS_MEMORY_BUFFER_ALLOC_C
+    scripts/config.pl set MBEDTLS_MEMORY_BACKTRACE
```

Comment:
Also enable `MBEDTLS_MEMORY_DEBUG` so that it's tested somewhere.

Reply:
Addressed in 125c871.